recipe/meta.yaml

```diff
@@ -18,21 +18,28 @@ requirements:
   host:
     - python
     - pip
-    - setuptools_scm >=3.4
-    - setuptools >=42
+    - setuptools_scm
+    - setuptools
     - toml
     - wheel
   run:
     - python
     - colorama  # [win]
-    - importlib_resources # [py<37]
+  run_constrained:
+    - slack-sdk
+    - requests
+    - ipywidgets >=6
 
 test:
   requires:
+    - dask
+    - keras
+    - numpy
     - pip
     - pytest
     - pytest-timeout
     - pytest-asyncio
+    - rich
   source_files:
     - tests
     - setup.cfg
```

Comment:
Just a question: how are setup.cfg and the toml files used in tests?

Reply:
setup.cfg is required for the event_loop test.

Reply:
There was an issue with pytest-asyncio and its own event_loop test. It can be related https://github.com/AnacondaRecipes/pytest-asyncio-feedstock/pull/10/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR37 but I'm not 100% sure